controllers/configmap.go

```diff
@@ -6,24 +6,29 @@ import (
 	"hash/fnv"
 	"strconv"
 
-	envoyv2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
 	envoycorev2 "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
-	envoylistener "github.com/envoyproxy/go-control-plane/envoy/api/v2/listener"
-	accesslogv2 "github.com/envoyproxy/go-control-plane/envoy/config/accesslog/v2"
-	bootstrapv2 "github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v2"
-	accesslogfilterv2 "github.com/envoyproxy/go-control-plane/envoy/config/filter/accesslog/v2"
-	tcpproxyv2 "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/tcp_proxy/v2"
-	udpproxyv2 "github.com/envoyproxy/go-control-plane/envoy/config/filter/udp/udp_proxy/v2alpha"
+	accesslogfilterv3 "github.com/envoyproxy/go-control-plane/envoy/config/accesslog/v3"
+	bootstrap "github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v3"
+	envoyv3 "github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3"
+	envoycorev3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	envoyendpoint "github.com/envoyproxy/go-control-plane/envoy/config/endpoint/v3"
+	envoylistener "github.com/envoyproxy/go-control-plane/envoy/config/listener/v3"
```

Comment:
Bit of a nitpick and the original code doesn't do this well at all, but the convention we've generally used for aliasing for Envoy & K8s imports is to use the last word & version, e.g. core/v2 -> corev2


```suggestion
	corev2 "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
	accesslogv3 "github.com/envoyproxy/go-control-plane/envoy/config/accesslog/v3"
	bootstrapv3 "github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v3"
	clusterv3 "github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3"
	corev3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
	endpointv3 "github.com/envoyproxy/go-control-plane/envoy/config/endpoint/v3"
	listenerv3 "github.com/envoyproxy/go-control-plane/envoy/config/listener/v3"
```

Reply:
My worry here is below we import K8s resources as `corev1` which could get confusing between the two different packages?

Reply:
Ooops, I didn't spot that - good point